Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsViewModel.kt

```diff
@@ -88,11 +88,10 @@ class PersonDetailsViewModel @AssistedInject constructor(
     private suspend fun assembleList(personCertificates: PersonCertificates, isLoading: Boolean) =
         mutableListOf<CertificateItem>().apply {
             val priorityCertificate = personCertificates.highestPriorityCertificate
-
-            when {
-                priorityCertificate.isValid -> colorShade
-                else -> PersonColorShade.COLOR_INVALID
-            }.also { colorShadeData.postValue(it) }
+            val colorState = if (priorityCertificate.isValid || priorityCertificate is TestCertificate) {
```

Comment:
Invalid Test Certificate should be displayed as invalid , right? we should just exclude expiring and expiring_soon states of the test certificate

Reply:
in 8886 , the tester mentioned that it is not expected for expired TC to show as `invalid` ,but it does not mean we should not display invalid TC as invalid